### PR TITLE
Added Github Action

### PR DESCRIPTION
**Pull request by thedevnisarg**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -11,7 +11,7 @@
 .mtj.tmp/
 
 # Package Files #
-*.jar
+# *.jar
```
Comment (adesjardin) [inline .gitignore:14]: Shouldn't jar files still be excluded from git commits?

File: version.properties [hunk 2]
```diff
@@ -0,0 +1,6 @@
+version.buildmeta=
+version.major=1
+version.minor=0
+version.patch=1
+version.prerelease=SNAPSHOT
+version.semver=1.0.1-SNAPSHOT
```
Comment (adesjardin) [inline version.properties:6]: We should be on 1.0.0-SNAPSHOT right now, there are no previous releases of the linter

**Review by adesjardin — CHANGES_REQUESTED** [review]
Just a few small things to correct

File: .gitignore (hunk 1 above)
Comment (thedevnisarg) [reply .gitignore]: Added back in.

File: version.properties (hunk 2 above)
Comment (thedevnisarg) [reply version.properties]: changed.

**Review by adesjardin — APPROVED** [review]

**Pull request merged**